python/cudf/cudf/__init__.py

```diff
@@ -113,8 +112,7 @@
 rmm.register_reinitialize_hook(clear_cache)
 
 
-__version__ = get_versions()["version"]
-del get_versions
+__version__ = "23.04.00"
```

Comment:
Out of scope for this PR, but do you think it's possible that there can be _exactly one_ place where this version number lives? I see there are scripts for updating the versions in all possible places, it would be nice if somehow the `setup.py/__init__.py` and `CMakeLists.txt` could programmatically read from (say) `version.cfg` in the root directory.